docs/root/intro/version_history.rst

```diff
@@ -15,6 +15,7 @@ Version history
 * http: fixed a bug where large unbufferable responses were not tracked in stats and logs correctly.
 * http: fixed a crashing bug where gRPC local replies would cause segfaults when upstream access logging was on.
 * http: mitigated a race condition with the :ref:`delayed_close_timeout<envoy_api_field_config.filter.network.http_connection_manager.v2.HttpConnectionManager.delayed_close_timeout>` where it could trigger while actively flushing a pending write buffer for a downstream connection.
+* initial healthcheck jitter: added :ref:`initial jitter <envoy_api_field_core.HealthCheck.initial_jitter>` to add jitter to the first healthcheck in order to prevent a thundering herd on Envoy startup.
```

Comment:
nit: "health check: " (and reorder) (also check to see what this is called elsewhere but I think that's the title we have used)

Reply:
`health check` is used elsewhere, will fix. No reordering needed as this line is between h and j?

Reply:
Technically 'e' comes before 't' :)

Reply:
Did you mean reorder wording of title to `health check initial jitter`, per other health check comments, and then put before `http*`?

Reply:
I mean it should read "health check: ..." like other examples.

Reply:
done